README.md

```diff
@@ -1,26 +1,66 @@
-# Shared resources for Hypothesis front-end applications
+# `@hypothesis/frontend-shared`
```

Comment:
Perhaps we should consider renaming this package given that it now consists almost entirely of UI components.

Reply:
Yes, I'd like to consider this!